docs: improve wallet docs (#2154)
FuelLabs/fuels-ts: apps/docs/src/guide/wallets/connectors.md

# Connectors

`Fuel Wallet Connectors` provide a standardized interface to integrate multiple wallets with your DApps.

> By using these connectors, developers can simplify the process of wallet integration, enabling smooth user interactions within their applications.

Reviewer: ```suggestion
By using these connectors, developers can simplify wallet integration and enable smooth user interactions within their applications.
```
